documents/ogc-submission.adoc

```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
 
```

Comment:
... will be valid ...

Reply:
That's originating from the OGC template, I'm not sure whether we should change that.

Reply:
Ah, right. So, let's scratch that one.